bench-tps/src/bench.rs

```diff
@@ -348,8 +365,22 @@ fn generate_txs(
     dest: &VecDeque<&Keypair>,
     threads: usize,
     reclaim: bool,
+    old_blockhash: &mut Hash,
+    last_blockhash_time: &mut Instant,
 ) {
     let blockhash = *blockhash.read().unwrap();
+    if *old_blockhash != blockhash {
+        *old_blockhash = blockhash;
+        datapoint_info!(
+            "blockhash_stats",
+            (
+                "time_elapsed_since_blockhash_refreshed",
+                last_blockhash_time.elapsed().as_millis(),
+                i64
+            )
+        );
+        *last_blockhash_time = Instant::now();
+    }
```

Comment:
https://github.com/solana-labs/solana/blob/7af269ce0b6df746c177d89c27efcc4ade804c3e/bench-tps/src/bench.rs#L445-L446 This seems like a more natural place to add this logging?

Reply:
you mean this one? https://github.com/solana-labs/solana/blob/7af269ce0b6df746c177d89c27efcc4ade804c3e/bench-tps/src/bench.rs#L467-L474